README.mkd

```diff
@@ -7,48 +7,65 @@
 which provides support for expanding abbreviations similar to
 [emmet](http://emmet.io/).
 
+<div align="center">
+
 ![screenshot](https://raw.githubusercontent.com/mattn/emmet-vim/master/doc/screenshot.gif)
+</div>
```

Comment:
Thanks. Please remove this div tag. I don't want html tags in markdown. I'll merge this PR.